docs/manifest.json

```
@@ -261,7 +261,11 @@
             },
             {
               "title": "Middleware (Beta)",
-              "path": "/docs/middleware.md"
+              "path": "/docs/middleware.md",
+              "redirect": {
+                "destination": "/docs/advanced-features/middleware.md",
```

```suggestion
                "destination": "/docs/advanced-features/middleware",
```
The `.md` extension isn't needed for redirects